configuration-sample/ods-provisioning-app-dev.env.sample

```diff
@@ -41,3 +41,6 @@ PROV_APP_LOG_LEVEL_OPENDEVSTACK=DEBUG
 
 # Jenkins pipeline trigger secret (raw)
 PROV_APP_PIPELINE_TRIGGER_SECRET=changeme
+
+# Log files system location
+LOGGING_FILE_PATH=./logs/changeme
```

Comment:
Isn't there a better default that does not necessarily need to be changed?

Reply:
mmm...not sure what you mean... assuming that one of the roles of this example configuration is to highlight the properties that are environment specific and need to be changed, we can put something like ```/opt/changeme```, looks better for you?

Reply:
@michaelsauter did you mean to add the real path that we use in our prov app deployments? We can add it to minimize the customization of this file but because this is a sample file I thought the value above is good enough... should we change it for the real file path... what do yo think?

Reply:
Yes I would use the actual path (basically what was hardcoded before). If I get this change correctly, then one can now change the path e.g. for local development, but one does not have to change it for a typical ODS installation, right?

Reply:
more or less... local environment is using the project application.properties which default to ```./build/logs```. Assuming this file is not just only an example but a kind of template I will add the actual path here.

Reply:
@michaelsauter changes done